tasks/mx/inegi.py

```diff
@@ -216,19 +221,23 @@ def columns(self):
             weight=0,
             targets={geom: GEOM_REF},
         )
-        name = OBSColumn(
-            type='Text',
-            weight=0,
-        )
+        cols = OrderedDict([('the_geom', geom),
+                            ('cvegeo', geom_ref)
+                        ])
+
+        if self.resolution in RESPROPNAMES:
+            cols['nomgeo'] = OBSColumn(
+                id=self.resolution + '_name',
+                type='Text',
+                weight=1,
+                name= 'Nombre de {}'.format(self.resolution),
```

Comment:
We're using English for our Mexican measures, so we should change this to "Name of".